introduction.md

```diff
@@ -0,0 +1,3 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed porta nibh ut nunc vulputate, a tristique felis tincidunt. Morbi ultrices, mi id iaculis varius, lectus mi faucibus nibh, ut posuere sem dolor et dui. Aenean gravida pellentesque lorem ut laoreet. Nullam eros mauris, consectetur in consectetur ac, ultrices non odio. Aenean ultricies nisi ac libero euismod tincidunt vitae et neque. Praesent gravida elementum dui nec sollicitudin. Integer sit amet velit in eros pulvinar condimentum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Integer eu odio nec augue viverra posuere nec quis enim.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.